control-plane/helper/mutating-webhook-configuration/mutating_webhook_configuration.go

```diff
@@ -0,0 +1,51 @@
+package mutatingwebhookconfiguration
```

Comment:
this is repeated code that existed in and has been refactored out to here:
- control-plane/subcommand/controller/command.go
- control-plane/subcommand/inject-connect/command.go
- control-plane/subcommand/webhook-cert-manager/command.go